### PR TITLE
Added Countdown section to the Hero

### PR DESCRIPTION
**Pull request by draymond63**
As the title suggests, I've added the countdown section to the hero! The logic is pretty simple, but please do check it out. 

The only concern is the position over the Hero image; if anybody thinks they know a better way to position it over the image (i.e. more consistent in its relative position) please give it a try and push a commit to this branch!

image
Desktop view ^^

image
Phone view ^^

### CODE REVIEW

File: src/components/homePage/Hero.vue [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-			<h1>Canadian Undergraduate Technology Conference</h1>
-			<!-- <h2>Building our Future</h2> -->
-			<div style="height: 1em"/>
+	</div>
+	<!-- TEXT COLUMN -->
+	<span>
+		<h3>Welcome to the 2021</h3>
+		<h1>Canadian Undergraduate Technology Conference</h1>
+		<!-- <h2>Building our Future</h2> -->
+		<div style="height: 1em"/>
```
Comment (sanaasy) [inline src/components/homePage/Hero.vue:12]: You could also use `<br />` to do this

File: src/components/homePage/Countdown.vue [hunk 2]
```diff
@@ -1,29 +1,70 @@
 <template>
 	<div id="countdown">
-		{{ timeStamp }}
+		<p class="title">COUNTDOWN!</p>
```
Comment (sanaasy) [inline src/components/homePage/Countdown.vue:3]: in the design, it says `CUTC COUNTDOWN!` and its also slightly smaller than the numbers.

**Review by sanaasy — APPROVED** [review]
Overall, this looks like a good way of implementing the countdown!

**Comment by Adib234** [comment]
Great job!

**Review by Adib234 — APPROVED** [review]

File: src/components/homePage/Countdown.vue (hunk 2 above)
Comment (draymond63) [reply src/components/homePage/Countdown.vue:3]: True, probably something that should be changed tbh. Thanks!

File: src/components/homePage/Hero.vue (hunk 1 above)
Comment (draymond63) [reply src/components/homePage/Hero.vue:12]: Ah, facts. This is from a while ago :)

**Pull request merged**